extensions/logging/README.md

```diff
@@ -0,0 +1,158 @@
+# @loopback/extension-logging
+
+This module contains a component provides logging facilities based on
+[Winston](https://github.com/winstonjs/winston) and
+[Fluentd](https://github.com/fluent/fluent-logger-node).
+
+## Stability: :warning:Experimental:warning:
+
+> Experimental packages provide early access to advanced or experimental
+> functionality to get community feedback. Such modules are published to npm
+> using `0.x.y` versions. Their APIs and functionality may be subject to
+> breaking changes in future releases.
+
+## Architecture overview
+
+![logging-component](logging-component.png)
+
+## Installation
+
+```sh
+npm install --save @loopback/extension-logging
+```
+
+## Basic use
+
+The component should be loaded in the constructor of your custom Application
+class.
+
+Start by importing the component class:
+
+```ts
+import {LoggingComponent} from '@loopback/extension-logging';
+```
+
+In the constructor, add the component to your application:
+
+```ts
+this.component(LoggingComponent);
+```
+
+Now your application can add a controller as follows to leverage the logging
+facilities:
+
+```ts
+import {inject} from '@loopback/context';
+import {Logger} from '@loopback/extension-logging';
+
+class MyController {
+  // Inject a winston logger
+  @inject(LoggingBindings.WINSTON_LOGGER)
+  private logger: Logger;
```

Comment:
Maybe we should rename the interface to `WinstonLogger` if we want to make it more clear that our Logger has exactly the same interface as provided `Winston`?